cli/flox/src/commands/auth.rs

```diff
@@ -111,7 +111,11 @@ pub async fn authorize(client: BasicClient, floxhub_url: &Url) -> Result<Credent
             }
             .checkpoint()?;
 
-            let url = details.verification_uri().url().as_str();
+            let url = details
+                .verification_uri_complete()
+                .unwrap()
```

Comment:
Should either use `?` to bubble up the error, or add a comment explaining why we know it can be unwrapped
```suggestion
                .ok_or(anyhow!("something descriptive"))?
```

Reply:
Updated to use `expect` because the Auth0 API will always include this field (the oauth library doesn't know that). I also refactored out some of the duplication so we only look up that value in one place.